packages/@dcl/ecs/src/systems/events.ts

```diff
@@ -110,14 +111,15 @@ export function createPointerEventsSystem(engine: IEngine, inputSystem: IInputSy
   }
 
   function setPointerEvent(entity: Entity, type: PointerEventType, opts: EventSystemOptions) {
-    if (opts.hoverText || opts.showFeedback) {
+    if (opts.hoverText || opts.showFeedback || opts.showHighlight) {
```

Comment:
is this necessary ? not sure why we have this so asking just in case

Reply:
great catch! by removing that check we solve an existent bug (when using `pointerEventsSystem.onPointerDown()` without any of the pointer event optional properties)

Reply:
tackled at https://github.com/decentraland/js-sdk-toolchain/pull/1024/commits/0abea71304fb5c50ecd1ebbfc4af91a83673cd83